kuksa_apps/node-red/README.md

```diff
@@ -22,26 +28,31 @@ wget https://github.com/eclipse/kuksa.val/releases/download/0.1.8/kuksa-val-0.1.
 docker load -i kuksa-val-0.1.8-arm64.tar.xz
 ```
 
-Then start all needed container using [`docker-compose.yml`](./docker-compose.yml):
+After that, just start all needed container using [`docker-compose.yml`](./docker-compose.yml):
 ```
-sudo apt  install docker-compose
+sudo apt install docker-compose
 ARCH=arm64 docker-compose up -d
 ```
 `-d` for detach.
 
-By default, the mqtt flows will be configured in node-red. You can also use the import feature inside node-red to try the websocket example [below](#Websocket). 
+**Note**: You need to be in this [directory](./) to be able to start the [`docker-compose.yml`](./docker-compose.yml) to use `docker-compose` to start all docker containers or shut down all docker containers:
```

Comment:
@anntexx I add this **Note**. Is it then clear? I think, it is little bitte confuse to add `cd`. if the user is already in this directory.

Reply:
If it is ok, @anntexx you can just approve the pr and then I can merge it.